Library/Homebrew/dev-cmd/ruby.rb

```diff
@@ -30,6 +30,7 @@ def ruby
 
     begin
       safe_system RUBY_PATH,
+                  "-W0",
```

Comment:
```suggestion
                  "ENV["HOMEBREW_RUBY_WARNINGS"]",
```

Alternatively, set it as a constant somewhere. Might be nice to replace all other uses of `-W0` with this, too?

Reply:
Replaced all occurrences I've found.